dapp/tests/e2e/app.test.js

```diff
@@ -65,6 +65,33 @@ describe('sendEmail', () => {
       Error('Failed to parse options from requester secret')
     );
   });
+  it('should fail if IEXEC_REQUESTER_SECRET_3 (options) contains invalid content-type', async () => {
+    process.env.IEXEC_REQUESTER_SECRET_3 = '{"contentType":"notacontenttype"}';
+    await expect(() => start()).rejects.toThrow(
+      Error('"contentType" must be one of [text/plain, text/html]')
+    );
+  });
+  it('should fail if IEXEC_REQUESTER_SECRET_3 (options) senderTag is empty', async () => {
+    process.env.IEXEC_REQUESTER_SECRET_3 = '{"senderTag":""}';
+    await expect(() => start()).rejects.toThrow(
+      Error('"senderTag" is not allowed to be empty')
+    );
+  });
+  it('should fail if IEXEC_REQUESTER_SECRET_3 (options) senderTag lenght is less than 3 characters', async () => {
```

Comment:
typo

```suggestion
  it('should fail if IEXEC_REQUESTER_SECRET_3 (options) senderTag length is less than 3 characters', async () => {
```

I suggest using [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) extension in VSCode